app/views/users/_user_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_tag path, method: method do %>
+  <%= label_tag :name %>
+  <%= text_field_tag :name, (if @new_user.present? then @new_user.name else nil end) %>
```

Comment:
There has to be a way to DRY this up because it seems redundant to use this if...else logic for every single part of the form we want pre-populated.